llvm/lib/Target/X86/X86AsmPrinter.cpp

```diff
@@ -55,6 +55,116 @@ X86AsmPrinter::X86AsmPrinter(TargetMachine &TM,
 // Primitive Helper Functions.
 //===----------------------------------------------------------------------===//
 
+const TargetInstrInfo *TII;
+
+/// Clear any mappings that map to the given register.
+void clearRhs(Register Reg, std::map<Register, int64_t> &SpillMap) {
+  for (auto I = SpillMap.begin(); I != SpillMap.end();) {
+    if (I->second == Reg) {
+      I = SpillMap.erase(I);
+    } else {
+      ++I;
+    }
+  }
+}
+
+/// Given a MachineBasicBlock, analyse its instructions to build a mapping of
+/// where duplicate values live. This can be either in another register or on
+/// the stack. Since registers are always positive and stack offsets negative,
+/// we can store both in the same map while still being able to distinguish
+/// them. This allows us to keep the changes in Stackmaps.cpp to a minimum and
+/// avoids having to alter the stackmap format.
+void processInstructions(
+    const MachineBasicBlock *MBB,
+    std::map<Register, int64_t> &SpillMap,
+    std::map<const MachineInstr *, std::map<Register, int64_t>> &StackmapSpillMaps
+  ) {
+  for (const MachineInstr &Instr : MBB->instrs()) {
+
```

Comment:
Extraneous blank line (oh how I miss those words!).

Reply:
Check this out: [1d5124c](https://github.com/ykjit/ykllvm/pull/57/commits/1d5124c610dcde8bbe12aeb77ba7e2d78e71338f)